src/services/users/index.js

```diff
@@ -152,11 +153,38 @@ const getNewTokens = async (email, currentRefreshToken) => {
 	return { accessToken, refreshToken, userId: user.id }
 }
 
+const getInvitedUserDetail = async (token) => {
+	try {
+		const decoded = jwt.verify(token, config('TOKEN_SECRET'))
+		const {email} = decoded
+		const result = await  db.query(
+			`SELECT * FROM user_invites
+		WHERE email = $1 
+		`,
+			[email]
+		)
+		const user = result.rows[0]
```

Comment:
if the invited user does not exist, kindly throw the appropriate error

if the invited user is already active, kindly throw the appropriate error

Reply:
okay